notes/CodeFlag_notes.csv

```diff
@@ -176,4 +176,5 @@ noteID,note
 179,"BFI2D is calculated by dividing the BFI by an expression that increases with the directional spread and decreases with the peakedness. It comes from the consideration that an increase of the directional spread reduces the excess of kurtosis, which is not taken into account in the BFI alone. Mori, N., M. Onorato, and P. A. E. M. Janssen, 2011: On the Estimation of the Kurtosis in Directional Sea States for Freak Wave Forecasting. J. Phys. Oceanogr., 41, 1484–1497, https://doi.org/10.1175/2011JPO4542.1"
 180,"Crest-trough correlation is an estimate of the auto-correlation between the crest heights and the trough depths. In narrow-banded seas this parameter is quite high, because crests and troughs are approximately the same size. Gemmrich, J., Cicon, L. Generation mechanism and prediction of an observed extreme rogue wave. Sci Rep 12, 1718 (2022). https://doi.org/10.1038/s41598-022-05671-4"
 181,"This is a metric that quantifies the wave-induced mechanical fatigue on the sea-ice pack (0 means broken, 1 means solid). This metric only makes sense in the presence of sea ice."
-182,"The x- and y- components of the wave radiative stress to sea-ice are not necessarily equivalent to the u- and v- components (eastward/northward). The x- and y- components strictly follow the defined coordinate system which may or may not follow the eastward and northward directions."
\ No newline at end of file
+182,"The x- and y- components of the wave radiative stress to sea-ice are not necessarily equivalent to the u- and v- components (eastward/northward). The x- and y- components strictly follow the defined coordinate system which may or may not follow the eastward and northward directions."
+317a,"The atmospheric surface momentum flux is what the atmosphere is losing at the surface of the ocean. The surface momentum flux into the ocean is the momentum flux that is entering the ocean and driving the ocean circulation. The difference is that part of the atmospheric momentum flux goes into the waves and not into the ocean circulation or part of the momentum contained in the wave field that is transferred to the ocean when waves break. In short, the ocean surface has waves which modulate the momentum transfer between the atmosphere and the ocean interior."
```

Comment:
```suggestion
317a,"The atmospheric surface momentum flux is what the atmosphere is losing at the surface of the ocean. The surface momentum flux into the ocean is the momentum flux that is entering the ocean and driving the ocean circulation."
```

Reply:
@sebvi @SibylleK I removed last part of the note because I think it added confusion and the first part is clear enough. Will you please review and confirm that this is ok?